HTMessaging/Classes/Connection.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Connection.swift
+//  HTMessaging
+//
+//  Created by Hector Jose Alvarado Chapa on 22/12/19.
+//  Copyright © 2019 Hector Jose Alvarado Chapa. All rights reserved.
+//
+
+import Foundation
+import Firebase
+
+public final class Connection {
+    func withFirebaseFile(with fileName: String, ofType: Filetype, completion: @escaping(_ status: FIRResponse?) -> Void) {
```

Comment:
Can you add a MARK here? Maybe one that reads: "Initialization"

Reply:
Can you add  javadocs like comments?

Like this:

/**
 *  Performs a connection to SendBird with the user ID.
 *
 *  @param userId            The user ID.
 *  @param completionHandler The handler block to execute. `user` is the object to represent the current user.
 */